src/bypass-device.cpp

```diff
@@ -0,0 +1,108 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#include "bypass-device.h"
+#include "stream.h"
+
+namespace librealsense
+{
+    bypass_device::bypass_device()
+        : device(std::make_shared<context>(backend_type::standard), {})
+    {
+        register_info(RS2_CAMERA_INFO_NAME, "Bypass-Sensor");
```

Comment:
Why not take the device name from the user?
Or better yet allow user to register its own info?